src/lib/net/src/utils/broadcast.rs

```diff
@@ -70,7 +70,7 @@ fn get_all_entities(state: &GlobalState) -> HashSet<Entity> {
     state
         .universe
         .get_component_manager()
-        .get_entities_with::<ChunkReceiver>()
+        .get_entities_with::<PlayerIdentity>()
```

Comment:
Keep this as a ChunkReceiver, since you don't want to send play packets to someone in login state.

Reply:
There has to be a better solution that that. Also why would a player during login have a player identity?

Reply:
You're more than welcome to find a better solution, but, player receives the PlayerIdentity component during login start packet. 
Oh wait, you also add ChunkReceiver during login start? Add it when login process is completed, and keep this as is. So 'everyone' would be players that could receive chunks I suppose?